etc/generated_headers/bson-version.h

```diff
@@ -14,12 +14,6 @@
  * limitations under the License.
  */
 
-
-#if !defined (BSON_INSIDE) && !defined (BSON_COMPILATION)
-#error "Only <bson/bson.h> can be included directly."
```

Comment:
These aren't needed since nobody is importing header files directly right?

Reply:
correct, these are headers generated by the `configure` or cmake process